config/schedule.rb

```diff
@@ -43,20 +43,23 @@ def local(time)
   runner "DuplicatePassportAnalytics.report"
 end
 
-every :day, at: local("05:30 am"), roles: [:cron] do
-  if CountryConfig.current_country?("India") && SimpleServer.env.production?
-    runner "ExperimentControlService.schedule_daily_stale_patient_notifications(name: 'Small Stale Patient July 2021', patients_per_day: 100)"
-  end
-end
-
 every :monday, at: local("6:00 am"), roles: [:cron] do
   if Flipper.enabled?(:weekly_telemed_report)
     rake "reports:telemedicine"
   end
 end
 
-every :day, at: local("07:00 am"), roles: [:cron] do
-  runner "AppointmentNotification::ScheduleExperimentReminders.perform_now"
+every :day, at: local("07:30 am"), roles: [:cron] do
+  if CountryConfig.current_country?("India") && SimpleServer.env.production?
+    runner "ExperimentControlService.start_current_patient_experiment(name: 'Current Patient August 2021')"
```

Comment:
I'm not worried about them running at the same time because the current job should do all its work on Wednesday and the stale job should do its work every day after.